tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+from typing import NamedTuple
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import audio_utils
+from tensorflow_lite_support.python.task.audio.core.pybinds import audio_buffer
+
+
+class AudioFormat(NamedTuple):
```

Comment:
@kinaryml Please use dataclasses to align with other API.

Reply:
@khanhlvg It's not being used and I forgot to remove the NamedTuple. I've fixed it now, thanks!